assets/pages/regular-contributions/helpers/ajax.js

```diff
@@ -184,11 +192,14 @@ function statusPoll(
   dispatch: Function,
   csrf: CsrfState,
   referrerAcquisitionData: ReferrerAcquisitionData,
+  paymentMethod: PaymentMethod,
+  participations: Participations,
 ) {
```

Comment:
I would add `void` as a return type to indicate that returns `undefined`

Reply:
I can't because it returns  Promise<Response> lower down

Reply:
It does have return-type `void` doesn't it? The only thing it does is call `setTimeout`, which implicitly returns `undefined`?

Reply:
@Ap0c this should be sorted now it returns ?Promise<void>

Reply:
@rupertbates nice! I think I got a bit lost in the diff here anyway, I mixed this up with `delayedStatusPoll` 🙃